x-pack/plugins/lens/public/editor_frame_plugin/plugin.tsx

```diff
@@ -8,10 +8,14 @@ import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
 import { I18nProvider } from '@kbn/i18n/react';
 import { CoreSetup } from 'src/core/public';
+import { HashRouter, Switch, Route, RouteComponentProps } from 'react-router-dom';
+import chrome from 'ui/chrome';
 import { DataSetup, ExpressionRenderer } from '../../../../../src/legacy/core_plugins/data/public';
 import { data } from '../../../../../src/legacy/core_plugins/data/public/setup';
 import { Datasource, Visualization, EditorFrameSetup, EditorFrameInstance } from '../types';
 import { EditorFrame } from './editor_frame';
+import { LensSavedObjectStore } from '../persistence/lens_store';
```

Comment:
Same thing about the naming here - could also be `SavedObjectStore` as it's part of lens and is not exported